src/colvar.cpp

```diff
@@ -1892,6 +1892,48 @@ int colvar::update_cvc_config(std::vector<std::string> const &confs)
 }
 
 
+int colvar::cvc_param_exists(std::string const &param_name)
+{
+  if (is_enabled(f_cv_single_component)) {
+    return cvcs[0]->param_exists(param_name);
+  }
+  return cvm::error("Error: calling colvar::cvc_param_exists() for a variable "
+                    "with more than one component.\n", COLVARS_NOT_IMPLEMENTED);
+}
+
+
+void const *colvar::get_cvc_param(std::string const &param_name)
```

Comment:
It seems it is easy to make it compatible with multiple components. For example:
```
void const *colvar::get_cvc_param(std::string const &param_name, size_t cvc_index = 0) {
  return cvcs[cvc_index]->get_param(param_name);
}
```

Reply:
Indeed, that would be the way to extend it easily. I would like to start with a single component first, make sure this interface does what is needed for Fabrizio's code, and extend it after that.